### PR TITLE
chore: update docs based on Helm revisions

### PR DESCRIPTION
**Pull request by khorne3**
- This PR makes no changes to workspace providers, since those were included in a separate PR
- Used https://github.com/cdr/enterprise-helm/blob/main/templates/NOTES.txt to help find areas of concern

### CODE REVIEW

**Comment by khorne3** [comment]
cc: @bpmct feel free to add to this or suggest changes!

File: admin/devurls.md [hunk 1]
```diff
@@ -15,43 +15,13 @@ dev URLs with Coder.
...
-
-```yaml
- - host: "*.my-custom-domain.io"
-    http:
-      paths:
-      - path: /
-        backend:
-          serviceName: envproxy
-          servicePort: 8080
+helm upgrade coder coder/coder --set --set devurlsHost="*.my-custom-domain.io"
```
Comment (khorne3) [inline admin/devurls.md:22]: this doesn't work

File: workspaces/workspace-templates/templates.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ workspace:
         scheme: http
         access: private
       - name: PublicPort
-        port: 8080
+        port: 80
```
Comment (khorne3) [inline workspaces/workspace-templates/templates.md:99]: @Emyrk can you please verify?

File: setup/air-gapped/index.md [hunk 3]
```diff
@@ -156,11 +135,9 @@ platform images are hosted in Coder's Docker Hub repo.
 
    ```bash
    helm install --wait --atomic --debug --namespace coder coder . \
-      --set cemanager.image=$REGISTRY_DOMAIN_NAME/coderenvs/coder-service:<version> \
-      --set envproxy.image=$REGISTRY_DOMAIN_NAME/coderenvs/coder-service:<version> \
+      --set coderd.image=$REGISTRY_DOMAIN_NAME/coderenvs/coder-service:<version> \
       --set envbox.image=$REGISTRY_DOMAIN_NAME/coderenvs/envbox:<version> \
       --set timescale.image=$REGISTRY_DOMAIN_NAME/coderenvs/timescale:<version> \
```
Comment (kylecarbs) [inline setup/air-gapped/index.md:140]: This should be `postgres.default.image` I believe.

File: workspaces/workspace-templates/templates.md (hunk 2 above)
Comment (kylecarbs) [reply workspaces/workspace-templates/templates.md:99]: This should be 443.

File: workspaces/workspace-templates/templates.md [hunk 4]
```diff
@@ -325,7 +325,7 @@ any dev URLs you create.
 dev-urls:
   value:
     - name: PublicPort
-      port: 8080
+      port: 80
```
Comment (kylecarbs) [inline workspaces/workspace-templates/templates.md:328]: Should also be 443.

**Review by kylecarbs — APPROVED** [review]
Just minor stuff... awesome changes!

**Pull request merged**